### PR TITLE
replaced runuser command by SU

### PR DESCRIPTION
**Pull request by rsantos88**
replaced runuser command by SU

### CODE REVIEW

File: share/teoBase/scripts/teoBase.xml [hunk 1]
```diff
@@ -38,14 +38,14 @@
     </module>
 
     <module>
-        <name>/sbin/runuser</name>
-        <parameters>-l teo -c "GST_PLUGIN_PATH=/usr/local/lib/gstreamer-1.0 speechRecognition.py"</parameters>
+        <name>su</name>
+        <parameters>- teo -c "GST_PLUGIN_PATH=/usr/local/lib/gstreamer-1.0 speechRecognition.py"</parameters> -->
```
Comment (PeterBowman) [inline share/teoBase/scripts/teoBase.xml]: Does this actually work on the YARP module manager? That closing `-->` looks like malformed XML, and the fully expanded command resolves to `su - teo -c ...` (note the stray `-` character).

Comment (rsantos88) [reply share/teoBase/scripts/teoBase.xml]: yes, it works. See https://askubuntu.com/a/606149
> `su - <username> -c "<commands>"`
> `-` will simulate a login of the specified user
> -c tells it that you want to run a command

Comment (jgvictores) [reply share/teoBase/scripts/teoBase.xml]: No offense to the non-draconian error treatment, but the trailing ` -->` should definitely be removed.

Comment (rsantos88) [reply share/teoBase/scripts/teoBase.xml]: Upss! Sorry I didn't see the end. Now it's fixed :)
Thanks.

**Review by jgvictores — APPROVED** [review]
Ok, cool, thanks!

**Pull request merged**